src/frequenz/client/microgrid/_client.py

```diff
@@ -48,33 +48,41 @@
 
 
 class ApiClient:
-    """Microgrid API client implementation using gRPC as the underlying protocol."""
+    """A microgrid API client."""
 
     def __init__(
         self,
-        grpc_channel: grpclib.client.Channel,
-        target: str,
+        server_url: str,
+        *,
```

Comment:
Should we need to explicitly clarify the `ApiClient` no longer needs the grpc_channel parameter to be constructed?

Reply:
Not sure what do you mean specifically, the release notes currently say:

- The client now uses a string URL to connect to the server. The current accepted format is `grpc://hostname[:<port:int=9090>][?ssl=<ssl:bool=false>]`, meaning that the `port` and `ssl` are optional and default to 9090 and `false` respectively. You will have to adapt the way you connect to the server in your code.

Isn't that saying what you are asking for? You mean saying explicitly that the argument `target` and `grpc_channel` were merged into `server_url`?

Reply:
Updated.